src/logging.cc

```diff
@@ -628,7 +628,7 @@ inline void LogDestination::RemoveLogSink(LogSink *destination) {
   MutexLock l(&sink_mutex_);
   // This doesn't keep the sinks in order, but who cares?
   if (sinks_) {
-    for (int i = sinks_->size() - 1; i >= 0; i--) {
+    for (int i = (int)sinks_->size() - 1; i >= 0; i--) {
```

Comment:
But `ptrdiff_t` is fine. `ptrdiff_t` is actually a better choice, since the type is standardized in C++, whereas `ssize_t` is not.